vault/dendron.topic.graph-view.md

```diff
@@ -11,23 +11,26 @@ custom: {}
 
 Dendron provides support for graph view using a fork of the [markdown-links](https://marketplace.visualstudio.com/items?itemName=tchayen.markdown-links) extension. 
 
-The Dendron graph view is generated using our hierarichal index. It also shows different relationships than most of the graph views you're probably used to.
+The Dendron graph view is generated using our hierarchical index. It also shows different relationships than most of the graph views you're probably used to.
 
-Instead of showing edges as links between nodes, Dendron edges are the hierarichal relationships between your notes. Think of it as a 20,000 foot view of the structure of your knowledge base. 
+Instead of showing edges as links between nodes, Dendron edges are the hierarchical relationships between your notes. Think of it as a 20,000 foot view of the structure of your knowledge base. 
 
-Note that this command is not optimized for performance and you might notice slowdowns if you have more than a few hundred notes. If you are running into performance issues, please upvote this [issue](https://github.com/dendronhq/dendron/issues/107) to prioritize the performacne work!
+Note that this command is not optimized for performance and you might notice slowdowns if you have more than a few hundred notes. If you are running into performance issues, please upvote this [issue](https://github.com/dendronhq/dendron/issues/107) to prioritize the performance work!
 
 ## Commands
 
 ### Dendron: Show Note Graph
 
+Ctrl + Shift + P: `> Dendron: Show Note Graph`
```

Comment:
this is different depending on os. i would say 'launch the command bar' with a link to [this](https://www.dendron.so/notes/401c5889-20ae-4b3a-8468-269def4b4865.html#launch-the-command-bar)